nearlib/near.js

```diff
@@ -9,6 +11,17 @@ class Near {
         this.nearClient = nearClient;
     }
 
+    /**
+     * Default setup for browser
+     */
+    static createDefaultConfig() {
+        //.keyStore, nearConnection
```

Comment:
hmm? is this trying to comment constructor params? in that case we can just pass object like

```js
new NearClient({
   keyStore: ...,
   connection: ...
});
```